### PR TITLE
edit next-js-simple readme

### PR DESCRIPTION
**Pull request by kapunahelewong**
## Description

This PR edits and builds upon the `next-js-simple` readme and provides detailed instructions with screenshots on how to use our example next.js app with Builder.

cc @steve8708 @mrkoreye

### CODE REVIEW

**Review by mrkoreye — APPROVED** [review]
Awesome! 🎉 

This looks great.

**Comment by mrkoreye** [comment]
FYI anyone else reviewing you can preview it here: https://github.com/BuilderIO/builder/blob/b2074aaa06fe631edea3f5ce4f1d114d7c69a10a/examples/next-js-simple/README.md

File: examples/next-js-simple/README.md [hunk 1]
```diff
@@ -1,45 +1,147 @@
...
 
 ### Deploy
 
 You can deploy anywhere you like, but for this project we recommend [Vercel](https://nextjs.org/docs/deployment).
 [![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https%3A%2F%2Fgithub.com%2Fbuilderio%2Fbuilder%2Ftree%2Fmaster%2Fexamples%2Fnext-js-simple)
+
+
+## Next steps
+
+* For more information on previewing your applications, see [Editing and previewing directly on your site](https://www.builder.io/c/docs/guides/preview-url).
```
Comment (steve8708) [inline examples/next-js-simple/README.md:145]: This readme is incredible. Only suggestion is lets point people how to register their react components too. Please feel free to change copy

```suggestion
* Learn how to [use your react components in our visual editor](https://www.builder.io/c/docs/custom-react-components)
* For more information on previewing your applications, see [Editing and previewing directly on your site](https://www.builder.io/c/docs/guides/preview-url).
```

**Review by steve8708 — APPROVED** [review]

File: examples/next-js-simple/README.md (hunk 1 above)
Comment (kapunahelewong) [reply examples/next-js-simple/README.md:145]: Meant to note this upon submission: resolved in https://github.com/BuilderIO/builder/pull/558

**Pull request merged**